recipes/libgettext/all/conanfile.py

```diff
@@ -26,7 +26,7 @@ class GetTextConan(ConanFile):
     topics = ("gettext", "intl", "libintl", "i18n")
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://www.gnu.org/software/gettext"
-    license = "GPL-3.0-or-later"
```

Comment:
Maybe we can add a comment here, to be aware of the different licenses in case we decide to change it in the future?

Reply:
Done.